packages/flutter/lib/src/material/page.dart

```diff
@@ -179,4 +183,12 @@ class MaterialPageRoute<T> extends PageRoute<T> {
 
   @override
   String get debugLabel => '${super.debugLabel}(${settings.name})';
+
+  String _routeName() {
+    if (_useCupertinoTransitions)
```

Comment:
On iOS you can never provide a name to be announced for routes. Is that intentional?

Reply:
You can override this behavior by providing a child route semantics with a name.  The default is none since that reflects the behavior of accessibility in the iOS apps I have seen